apps/server/lib/lexical/server/code_intelligence/entity.ex

```diff
@@ -0,0 +1,142 @@
+defmodule Lexical.Server.CodeIntelligence.Entity do
+  alias Future.Code, as: Code
+  alias Lexical.Ast
+  alias Lexical.Document
+  alias Lexical.Document.Location
+  alias Lexical.Document.Position
+  alias Lexical.Document.Range
+  alias Lexical.Project
+  alias Lexical.RemoteControl
+  alias Lexical.Text
+
+  require Logger
+
+  @type resolved :: {:module, module()}
```

Comment:
this means we only resolve modules presently, correct?

Reply:
Correct. The idea is that this will (quickly) expand as other entities are supported.